modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/utils/RestUtils.java

```diff
@@ -71,14 +72,21 @@
 
         Map<String, Object> release = new HashMap<>();
         release.put(SW360Attributes.RELEASE_COMPONENT_ID, componentId);
-        release.put(SW360Attributes.RELEASE_ID, sw360Release.getReleaseId());
+        if (sw360Release.getReleaseId() != null) {
```

Comment:
isn't this compulsory

Reply:
no, if you create a release you do not yet know the ID